src/reply.rs

```diff
@@ -287,6 +289,10 @@ impl ReplyOpen {
     /// you can pass it as the 3rd parameter of `OpenReply::opened_passthrough()`.  This is done in
     /// two separate steps because it may make sense to reuse backing IDs (to avoid having to
     /// repeatedly reopen the underlying file or potentially keep thousands of fds open).
+    /// # Errors
+    /// Propagates errors due to communicating with the fuse device.
+    /// # Panics
+    /// Panics if this reply object has already been used.
```

Comment:
How can this happen? All the other methods move self

Reply:
Currently, it cannot happen. However, it is conceivable that someday, it could become possible. FUSE might force upon us a requirement to support some funny sequence like "reply and then do something else that isn't a reply", that would make it possible to misuse these functions. 

The true, secret reason for this panic is that it isn't possible to define a "consumes self" method on a "box dyn" object. So we have to put it in an option so we can consume the option. Introducing the option introduces the panic. I have a plan in the works that will remove the box dyn from the object stack, which makes this entire train of thought irrelevant.

Reply:
Let's remove it then. We shouldn't document things that can't happen, and I generally don't add "Panics" documentation unless it should clearly be part of the public API

Reply:
Here's an example of how I configured clippy in another crate I maintain: https://github.com/cberner/redb/blob/198df7536cddde80fa7d83b2d80b142d7d89958b/src/lib.rs#L1-L17